internal/api/changeset_translator.go

```diff
@@ -235,3 +237,151 @@ func (t changesetTranslator) optionalFloat64(value *float64, field string) model
 
 	return models.NewOptionalFloat64Ptr(value)
 }
+
+func (t changesetTranslator) fileIDPtrFromString(value *string, field string) (*models.FileID, error) {
```

Comment:
While there is an argument to be made for consistency, I don't think it's worth keeping `field` in functions that don't use them.

Reply:
Yeah the reason I did that was for consistency, and because other functions there did just that - `string`, `datePtr`, `intPtrFromString`, etc.